pkg/process/checks/enable_checks_containerized_test.go

```diff
@@ -164,3 +167,26 @@ func createDeps(t *testing.T) deps {
 		}),
 	)
 }
+
+func assertContainsCheck(t *testing.T, checks []string, name string) {
```

Comment:
Copied over from `pkg/process/checks/enabled_checks_test.go`. These tests will also be moved over in a subsequent PR and these functions will be removed then.